### PR TITLE
New upstart script for head pc

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

File: upstart/cob-h2-start [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License LGPL for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License LGPL along with this program.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+#################################################################
+
+export ROS_MASTER_URI=http://myrobot-b1:11311
```
Comment (ipa-fxm) [inline upstart/cob-h2-start]: `myrobot`?

File: upstart/cob-h2-start [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+#source /opt/ros/indigo/setup.bash
+source /home/robot/git/care-o-bot/src/install/setup.bash
```
Comment (ipa-fxm) [inline upstart/cob-h2-start]: on the robot hw it would be `/u/robot`, no?

**Comment by ipa-fxm** [comment]
Do all component PCs have their own upstart script?
I thought just the base has one with the main `cob_bringup robot.launch`...

File: upstart/cob-h2-start (hunk 1 above)
Comment (ipa-nhg) [inline upstart/cob-h2-start]: yes, this script has to be still tested on the robot, if it works I will create an install script where the robot name will be automatically set.
For example: https://github.com/ipa320/setup_cob4/blob/master/upstart/upstart_install.sh#L11

**Comment by ipa-nhg** [comment]
At the moment only base pc has an upstart script,  this script waits until all the pcs are booted and launch the robot.launch.
The new setup of the robot unfortunately requires a Ubuntu 32 bits system on the head pc. In order to do not have conflicts with the compilation we decided don't setup the 32 bits pc as a NFS client of the base PC (64 bits). Therefore the PC has local accounts (/home/robot...) and requires an extra upstart script.

File: upstart/cob-h2-start (hunk 2 above)
Comment (ipa-nhg) [inline upstart/cob-h2-start]: No, see the https://github.com/ipa320/setup_cob4/pull/23#issuecomment-185625926

**Comment by ipa-nhg** [comment]
Test on the robot: OK

**Comment by ipa-fmw** [comment]
CR: OK

**Pull request merged**